google/cloud/bigtable/internal/defaults.cc

```diff
@@ -32,6 +32,8 @@ namespace internal {
 
 namespace {
 
+auto constexpr kBackoffScaling = 2.0;
```

Comment:
This is the only default that gets a constant? 🤷

Reply:
Yes. https://github.com/googleapis/google-cloud-cpp/blob/6d936867bdfc5cd3071e5cb35d9a0291d22a106a/generator/internal/option_defaults_generator.cc#L84-L128